bee-message/src/parents.rs

```diff
@@ -6,16 +6,18 @@
 use crate::{MessageId, MessageUnpackError, ValidationError};
 
 use bee_ord::is_unique_sorted;
-use bee_packable::{coerce::*, PackError, Packable, Packer, UnpackError, Unpacker};
+use bee_packable::{
+    coerce::*, error::UnpackPrefixError, BoundedU8, InvalidBoundedU8, PackError, Packable, Packer, UnpackError,
+    Unpacker, VecPrefix,
+};
 
 use alloc::vec::Vec;
-use core::{
-    convert::{Infallible, TryFrom, TryInto},
-    ops::RangeInclusive,
-};
+use core::convert::{Infallible, TryFrom, TryInto};
 
-/// The range representing the valid number of parents.
-pub const MESSAGE_PARENTS_RANGE: RangeInclusive<usize> = 1..=8;
+/// Minimum number of parents for a valid [`ParentsBlock`].
+pub const PREFIXED_PARENTS_LENGTH_MIN: u8 = 1;
+/// Maximum number of parents for a valid [`ParentsBlock`].
+pub const PREFIXED_PARENTS_LENGTH_MAX: u8 = 8;
 
 /// Minimum number of strong parents for a valid message.
 pub const MESSAGE_MIN_STRONG_PARENTS: usize = 1;
```

Comment:
Can you put `MIN` at the end for consistency ? Or `MIN` at the middle, not sure what we do elsewhere

Reply:
We actually don't need this constant at all -- strong parents count is already validated in `message.rs`.

Reply:
Should we remove it then ?

Reply:
I removed it but didn't save :trollface: Something is wrong with my brain this morning.